.gitignore

```diff
@@ -28,5 +28,3 @@ src/css/tailwind.css
 # src/**/*.module.css
 
 docs
-
-src/css/tailwind.src.module.css
```

Comment:
Is this removal correct? Just asking since it doesn't relate to issue topic

Reply:
I think it's not. I'll add it.